MPF.md

```diff
@@ -738,6 +738,22 @@ Example configuration via augments:
 
 The above configuration would result in exec_command being triggered at the top and half hour and sleeping for up to `splaytime` before agent execution.
 
+### Configure cf-execd runagent socket users
+
+On Enterprise hubs, access to cf-execd sockets can be configure a list of users who should be allowed by defining `vars.control_executor_runagent_socket_allow_users`. By default on Enterprise Hubs, `cfapache` is allowed to access runagent sockets.
```

Comment:
It can be configure**d** on all hosts, not just on Enterprise hubs. It's just the `cfapache` as default that is different on Enterprise hubs.

Reply:
Yeah, I worded it this way because in the MPF where the policy applies the ACLs to the directory and socket, that is only done on enterprise hubs. Yes, if you configure it via augments, cf-execd will set perms, but cf-agent won't be enforcing it unless it's an enterprise hub.